Sources/TensorFlow/Layers/Sequential.swift

```diff
@@ -47,8 +47,7 @@ import _Differentiation
 /// ````
 public struct Sequential<Layer1: Module, Layer2: Layer>: Module
```

Comment:
[Kokoro CI fails](https://source.cloud.google.com/results/invocations/5929c914-47ad-4e47-b54d-dc8ada826fb7/targets/s4tf%2Fapis%2Fubuntu%2Fpresubmit/log) (for `TENSORFLOW_USE_STANDARD_TOOLCHAIN=NO`):
```
/swift-apis/Sources/TensorFlow/Layers/Sequential.swift:68:1: error: type 'Sequential<Layer1, Layer2>.TangentVector' does not conform to protocol 'VectorProtocol'
}
^
Swift.VectorProtocol:2:20: note: protocol requires nested type 'VectorSpaceScalar'; do you want to add it?
    associatedtype VectorSpaceScalar : AdditiveArithmetic
                   ^
```

It may be possible to fix this by adding an explicit `typealias VectorSpaceScalar = Float` definition for `Sequential.TangentVector`.

Reply:
I tried doing this in https://github.com/tensorflow/swift-apis/pull/1178/commits/34fca6a40f3ed0db948a0c3e005c72b6fb636838 but it didn't work, so I'm not sure what to try next.

Does anyone else have ideas? @BradLarson @compnerd

One option is to bite the bullet and drop support for the old baked-in-`TensorFlow` toolchains (which is what's causing the failure). I think that's a good option if we and the community are all on-board with it. That also allows us to drop custom internal support for CI and building OSS toolchains - we can just use GitHub Actions, which is much simpler.